producer/kinesis.go

```diff
@@ -123,6 +134,8 @@ func (prod *Kinesis) Configure(conf core.PluginConfig) error {
 
 	prod.streamMap = conf.GetStreamMap("StreamMapping", "default")
 	prod.batch = core.NewMessageBatch(conf.GetInt("BatchMaxMessages", 500))
+	prod.recordMaxMessages = conf.GetInt("RecordMaxMessages", 1)
+	prod.delimiter = []byte(conf.GetString("RecordMessageDelimiter", "\n"))
```

Comment:
The old behavior is default due to RecordMaxMessages defaulting to 1. Will
change this to "" to match consumer. Perhaps I should log a warning if
RecordMaxMessages > 1 and this is still ""? because the consumer can't
split on "".

On Thursday, May 19, 2016, Arne Claus <notifications@github.com> wrote:

> In producer/kinesis.go
> <https://github.com/trivago/gollum/pull/22#discussion_r63846158>:
>
> > @@ -123,6 +134,8 @@ func (prod *Kinesis) Configure(conf core.PluginConfig) error {
> >
> >  	prod.streamMap = conf.GetStreamMap("StreamMapping", "default")
> >  	prod.batch = core.NewMessageBatch(conf.GetInt("BatchMaxMessages", 500))
> > +	prod.recordMaxMessages = conf.GetInt("RecordMaxMessages", 1)
> > +	prod.delimiter = []byte(conf.GetString("RecordMessageDelimiter", "\n"))
>
> default value should be "" to match consumer and make old behavior default
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/trivago/gollum/pull/22/files/a99564704a7b4be8b402a6bded94cdc3ee11ac3f#r63846158>
>